content/account/organizations.textile

```diff
@@ -0,0 +1,82 @@
+</aside>
+
+h2(#primary). Primary account
+
+The primary account is an organization's main account and holds the following privileges:
+
+* The highest level of access to the organization.
+* Ownership of all all accounts within the organization.
+* The default account used for provisioning.
+
+h2(#accounts). Create accounts in an organization
+
+Create accounts in an organization:
+
+* Open the *Account* navigation dropdown.
+** Click "Organization Accounts.":https://ably.com/organizations/any/accounts
+** Click *New account*.
+** Add an account name and *Create account*.
+
+h2(#scim). Provision users via SCIM
+
+<aside data-type='note'>
+<p>Google Workspace _alone_ does not natively support SCIM.</p>
+</aside>
+
+Manage access to multiple Ably accounts through a single identity provider. To enable this, configure both "SSO":/account/sso with your chosen identity provider and "SCIM":#SCIM. Once configured, Ably automatically provisions and deprovisions users with access to the Ably app in your identity provider, either individually or through assigned groups. New users are added to Ably's default provisioning account with the role of *Developer*.
+
+Ably only recognize one registered email domain per organization, unrecognize email domains will result in rejected provisioning attempts.
+
+Users provisioned through SCIM cannot modify their name or email address within Ably. All personal information updates must be made through your identity provider, and changes will sync to Ably on the next SCIM update cycle.
+
+The following steps outline the process for provisioning users through SCIM:
+
+* Configure "SSO":/account/sso by enabling and setting up SSO Ably and your identity provider.
+* Set up SCIM using Ably's SCIM configuration details and add them to your identity provider. Ensure the required user provisioning options are selected.
+
+Once SSO and SCIM are configured, users can be provisioned automatically, ensuring centralized access control, improved security, and reduced manual effort.
+
+h2(#manual). Provision users manually
+
+If your organization doesn't use SSO and SCIM, you can manually add users:
+
+* Open the *Account* navigation dropdown.
+** Select *Organization Users* to view all users in your organization.
+** Click *Invite new user* to open the *Invite a user to join* page.
+** Enter the users *Name* and *Email*.
+** Specify the user's required *Permissions* for the organization.
+** Assign the necessary *Roles* for the required *Account* access.
+** Click *Invite user via email* to complete the invitation.
+
+h2(#manage). Manage roles
+
+Manage user and group "roles":/account/users#roles across all or specific accounts within the organization. User and group roles include those assigned directly to the user and through the groups the user belongs to.Use the *Organization Users* page as a central point of control, rather than managing access individually within each account.
```

Comment:
```suggestion
Manage user and group "roles":/account/users#roles across accounts within your organization. User and group roles include those assigned directly to the user and through the groups the user belongs to. Use the *Organization Users* page as a central point of control, rather than managing access individually within each account.
```

Can you add a link to the organization users page with `/any/` in the URL?